networking/configuring-cluster-network-range.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="configuring-cluster-network-range"]
+= Configuring the cluster network range
+include::_attributes/common-attributes.adoc[]
+:context: configuring-cluster-network-range
+
+toc::[]
+
+As a cluster administrator, you can expand the cluster network range after cluster installation. You might want to expand the cluster network range if you need more IP addresses for additional nodes.
+
+For example, if you deployed a cluster and specified `10.128.0.0/19` as the cluster network range, you are limited to 16 nodes. You can expand that to 510 nodes by changing the CIDR mask on a cluster to `/14`.
+
+When expanding the cluster network address range, your cluster must use the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes network plugin]. Other network plugins are not supported.
+
+The following limitations apply when modifying the cluster network IP address range:
+
+- The CIDR mask size specified must always be smaller than the currently configured CIDR mask size, because you can only increase the number of IP addresses in an installed cluster
```

Comment:
because you can only increase IP space by adding more nodes in an installed cluster